### PR TITLE
Fix #655: remove 'Reset Feedback' button.

### LINKED ISSUE #655: "Reset Feedback" Button should be removed and "Reset Code" should be updated.
Currently, there's a concern that Reset Feedback isn't terribly useful and Reset Code is more of a "Start Over". We should update the UI to reflect this and have Start Over do the following:

- end a session
- clear LocalStorage

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: client/question/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -1214,22 +1209,12 @@ tie.directive('learnerView', [function() {
...
-        $scope.resetFeedback = function() {
+          // Clear the code and feedback from localStorage, so that it is not
+          // retrieved in the subsequent initialization.
           SessionHistoryService.reset();
 
-          // Only add intro message if TIE is iframed so that the feedback
-          // window is not completely empty after resetting.
-          if (ParentPageService.isIframed()) {
-            SessionHistoryService.addIntroMessageBalloon();
-          }
```
Comment (rabidbit) [inline client/question/components/LearnerViewDirective.js]: Is showing intro message done elsewhere or will be in a separate PR?

Comment (seanlip) [reply client/question/components/LearnerViewDirective.js]: It's done by session reset.
